content/quickstart/gcp-deploy.md

```diff
@@ -167,8 +167,8 @@ Select **Create and Continue**.
 {{<img src="quickstart/images/gcp-sa.png" alt="GCP service account creation screen" align="center" size="small" unBlur="true" >}}
 
 ##### Grant this service account access to project
-For the _CloudSQL as a service_ configuration the service account requires the roles:  
-**Cloud SQL Admin**  
+For the _CloudSQL as a service_ configuration the service account requires the role:  
```

Comment:
> For the _CloudSQL as a service_ configuration 
Why are we discussing `CloudSQL as a service` configuration in general documentation?